### PR TITLE
add bash functions

### PR DESCRIPTION
**Pull request by andriy-safe-ai**
Tests:
```
# installed llvm
$ nix-install llvm

# get package id
$ nix-list | grep llvm
> 0 flake:nixpkgs... ...-llvm-11.1.0

# tried upgrading
$ nix-upgrade 0

# deleted
$ nix-remove 0

### CODE REVIEW

File: playbooks/install_nix.yml [hunk 1]
```diff
@@ -106,6 +106,16 @@
               eval "$command"
           }
 
+          nix-remove() {
+              command="nix profile remove $1"
```
Comment (steven-basart) [inline playbooks/install_nix.yml:110]: This one needs be able to accept more than one.

File: playbooks/install_nix.yml [hunk 2]
```diff
@@ -106,6 +106,16 @@
...
+              eval "$command"
+          }
+
+          nix-upgrade() {
```
Comment (steven-basart) [inline playbooks/install_nix.yml:114]: Both of them actually.

**Review by steven-basart — CHANGES_REQUESTED** [review]
Make it like the install to allow removal of more than 1 item.

File: playbooks/install_nix.yml (hunk 1 above)
Comment (andriy-safe-ai) [reply playbooks/install_nix.yml:110]: Updated.

File: playbooks/install_nix.yml (hunk 2 above)
Comment (andriy-safe-ai) [reply playbooks/install_nix.yml:119]: Updated.

**Comment by steven-basart** [comment]
LGTM squash then merge

**Pull request merged**